packages/api/cms-api/src/index.ts

```diff
@@ -139,6 +139,7 @@ export { PageTreeNodeCategory, PageTreeNodeInterface, PageTreeNodeVisibility, Sc
 export { PageExists, PageExistsConstraint } from "./page-tree/validators/page-exists.validator";
 export { PublicUpload } from "./public-upload/entities/public-upload.entity";
 export { PublicUploadModule } from "./public-upload/public-upload.module";
+export { PublicUploadsService } from "./public-upload/public-uploads.service";
```

Comment:
Do we need a changeset for this?

Reply:
it's a new api people can/should use in projects, so yes

Reply:
2e5c0886f5c49d2b2dc4c33a34e534a31f6a5c65